AUTHORS.md

```diff
@@ -0,0 +1,22 @@
+# Below is a list of people and organizations that have contributed
+
+# to the Dataverse Frontend project. Names should be added to the list like so:
+
+#
+
+# Name/Organization <email address>
+
+#
+
+# Anyone who has contributed to the Dataverse Frontend project in any way (not
+
+# limited to submitting PRs) is welcome to submit a PR to add their
+
+# name to this file.
+
+#
+
+# Thanks to everyone for your contributions!
+
+Harvard University IQSS Team
+FirstName LastName <firstname.lastname@example.com>
```

Comment:
I'm just curious where this AUTHORS file idea came from. I do see Sinatra uses it according to https://opensource.guide/building-community/#share-ownership-of-your-project via https://stackoverflow.com/questions/18459504/what-is-the-origin-and-purpose-of-authors-md

I'm not opposed to it. Again, just curious.

There's also CODEOWNERS: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

And there's All Contributors, which @poikilotherm is playing around with here:

- https://github.com/gdcc/dvopa/issues/1

Reply:
I removed the file for now since it was just a template that needs to be filled with real information.

There's an issue to work on it:
- #306

However, at the moment, it doesn't seem like a critical file in the Frontend Guides, and we have some other issues related to the frontend guides in this Sprint with higher priority than the AUTHORS file.
- #315